gmat/tests/template_frames.script

```diff
@@ -0,0 +1,113 @@
+%----------------------------------------
+%--   GMAT validate frame conversions  --
+%----------------------------------------
+
+% ABOUT
+% -----
+%
+% The following script validates a particular
+% frame conversion between a bodyICRS frame
+% and its associated bobyFixed one.
+
+%----------------------------------------
+%---------- Coordinate Systems
+%----------------------------------------
+
+% Define the bodyICRS frame
+Create CoordinateSystem bodyICRF;
+GMAT bodyICRF.Origin = {{ body }};
+GMAT bodyICRF.Axes = ICRF;
+
+% Define the bodyFixed frame
+Create CoordinateSystem bodyFixed;
+GMAT bodyFixed.Origin = {{ body }};
+GMAT bodyFixed.Axes = BodyFixed;
+
+%----------------------------------------
+%---------- Spacecraft
+%----------------------------------------
+
+% Build spacecraft's orbit from vectors
+Create Spacecraft ss_0;
+GMAT ss_0.DateFormat = TTGregorian;
+GMAT ss_0.CoordinateSystem = bodyICRF;
+GMAT ss_0.DisplayStateType = Cartesian;
+GMAT ss_0.X = {{ rx }}; 
```

Comment:
The values used in here are any combination produced by {-1, 0, 1}. However, GMAT shows to perform very slowly with those although it finally converges.

Reply:
What units are these? Meters?

Reply:
Either way, we will use planet radiuses